### DISCUSSION TITLE
QUESTION | IPFS网关加載失敗

### DISCUSSION THREAD

**Discussion by ElliotCHEN37** [MCSLTeam/MCSL2#117] [提问区域]
### MCSL2 系统报告

MCSL2系统报告：
生成时间：2023-08-25 17:59:44
MCSL2版本：2.2.1.2
操作系统：Windows10 10.0.22621
架构：64bit
内存占用：178.89MB

### 你在哪方面遇到了问题？

MCSL2下载

### 说明

系統規格：Windows 11 Professional 22H2 build 22621.2215 64bit
軟體版本：2.2.1.2

在下載頁面不論如何都顯示“加載失敗。請嘗試刷新，如果任然失敗，請匯報此Bug。”網路也都沒有問題。嘗試過關閉Proxy，但是結果還是相同。

### 附件(可选)

<img width="960" alt="螢幕擷取畫面 2023-08-25 180658" src="https://github.com/MCSLTeam/MCSL2/assets/109361374/7ba4aa21-2529-46f7-8c43-e73306004cd9">

### 确认

- [X] 我确保我的Issue没有和别人的重复

**Comment by ElliotCHEN37** [comment]
已解決

使用github源代碼自行組建即可解決

**Comment by lxhtt** [comment]
这是由于IPFS网关无法使用导致的。可跟从主页公告修改。
2.2.1.2中，我们也已经更换了默认的IPFS网关，您只需要删除MCSL2/MCSL2_Config.json文件，再启动MCSL2即可。
